src/utils/degreeConversion.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given an angle between [-360, 360], place it in range [0, 360]
```

Comment:
*question*: why is this useful? Maybe add a comment in the documentation that explains why this is needed.
Why are a angles > 360 not allowed? You are using modulo anyway.